techniques/aria/ARIA1.html

```diff
@@ -113,22 +108,22 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
       
       
    <section id="description">
-      <h2>Description</h2>
-      <p>The purpose of this technique is to demonstrate how to use the WAI-ARIA <a href="https://www.w3.org/TR/wai-aria-1.2/#aria-describedby"><code class="language-html">aria-describedby</code></a> property to provide programmatically determined, descriptive information about a user interface element. The <code class="language-html">aria-describedby</code> property may be used to attach descriptive information to one or more elements through the use of an <code class="language-html">id</code> reference list. The <code class="language-html">id</code> reference list contains one or more unique element <code class="language-html">id</code>s.</p>
-      <p>Refer to <a href="https://www.w3.org/TR/html-aria/">ARIA in HTML</a> for information on how to provide WAI-ARIA States and Properties with HTML. WAI-ARIA States and Properties are compatible with other languages as well; refer to documentation in those languages.</p>
+      <h2>解説</h2>
+      <p>この達成方法の目的は、プログラムによる解釈がされる、ユーザインタフェース要素に関する説明的な情報を提供するために、どのように WAI-ARIA <a href="https://www.w3.org/TR/wai-aria-1.2/#aria-describedby"><code class="language-html">aria-describedby</code></a> プロパティを使用するかを示すことにある。<code class="language-html">aria-describedby</code> プロパティは、ID 参照リストの使用を通して一つ以上の要素の説明的情報と結びつけるために使用してもよいものである。<code class="language-html">ID</code> 参照リストは、一つ以上のユニークな要素 <code class="language-html">ID</code> を含む。</p>
+      <p>HTML とともに WAI-ARIA ステート及びプロパティを提供する方法については、<a href="https://www.w3.org/TR/html-aria/">ARIA in HTML</a> を参照のこと。WAI-ARIA ステート及びプロパティは他の言語とも互換性がある。詳しくはそれらの言語の説明文書を参照のこと。</p>
```

Comment:
「～を参照のこと。」については、「～を参照。」に統一します。

```suggestion
      <p>HTML とともに WAI-ARIA ステート及びプロパティを提供する方法については、<a href="https://www.w3.org/TR/html-aria/">ARIA in HTML</a> を参照。WAI-ARIA ステート及びプロパティは他の言語とも互換性がある。詳しくはそれらの言語の説明文書を参照。</p>
```